onnxruntime/core/providers/openvino/openvino_execution_provider.cc

```diff
@@ -411,7 +422,14 @@ static bool IsUnsupportedOpMode(const Node* node, const onnxruntime::GraphViewer
     // nGraph only supports constant shape input values
     const auto& shape_input = node->InputDefs()[1];
     return !graph_viewer.IsConstantInitializer(shape_input->Name(), true);
-  }
+  } else if (optype == "ArgMax" || optype == "ArgMin" ) {
+      
+    auto dtype = node->InputDefs()[0]->TypeAsProto()->tensor_type().elem_type();
+    if (!(dtype == ONNX_NAMESPACE::TensorProto_DataType::TensorProto_DataType_FLOAT ||
+          dtype == ONNX_NAMESPACE::TensorProto_DataType::TensorProto_DataType_BFLOAT16)) {
```

Comment:
We anyway don't support BFLOAT16. So just have it check for float

Reply:
done